blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/ScheduleContextClearFromRunningTest.java

```diff
@@ -35,10 +39,15 @@ public void setUp() throws GranularityException {
         coarserGran = gran.coarser();
         coarserSlot = coarserGran.slot(updateTime);
 
-        ctx = new ScheduleContext(now, shards);
+        ctx = new ScheduleContext(now, shards, new Clock() {
+            @Override
+            public Instant now() {
+                return new Instant(lastRollupTime);
```

Comment:
You do an override here, but mock in later test class. Curious, why two different ways?